src/pydantic_ome_ngff/tree.py

```diff
@@ -27,13 +27,13 @@ class Node(StrictBase):
 
 
 class Array(Node):
-    node_type: NodeType = Field("array", const=True)
+    node_type: NodeType = "array"
```

Comment:
API change: `const` is gone, error messages say "use literal". Possibly this doesn't want to be a NodeType any more, but a `Literal["array"]`?
https://docs.pydantic.dev/latest/migration/#changes-to-pydanticfield